src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfoOfT.cs

```diff
@@ -19,6 +19,10 @@ public abstract class JsonTypeInfo<T> : JsonTypeInfo
             base(type, options, dummy: false)
```

Comment:
Can we remove this constructor now?

Reply:
I think so, will do it

Reply:
pending change, will be included in next iteration